walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
I didn't see this function in the js file. 
Also name the function just getRandomQuote.

Reply:
getRandomQuoteUsingArrowFunctions was part of the starter code, we were supposed to swap out the functions as practice using arrow functions.

Reply:
you can remove the function if it is no longer used. Ok to leave it for now.